node/src/server-modules/GlideFt.ts

```diff
@@ -486,10 +471,8 @@ export class GlideFt {
             FtSearchOptions & {
                 limited?: boolean;
             },
-    ): Promise<FtProfileReturnType> {
-        const args: GlideString[] = ["FT.PROFILE", indexName];
-
-        args.push("SEARCH");
+    ): Promise<[FtSearchReturnType, Record<string, number>[]]> {
```

Comment:
```suggestion
    ): Promise<[FtSearchReturnType, Record<string, number>]> {
```
I think only one record is returned.
Please double check in tests, I afraid client returns `GlideRecord`, which we should downcast to a regular record (as we do in `xinfo` commands)